### PR TITLE
Import SV - change sql to insert or update study 

### PR DESCRIPTION
**Pull request by dglemos**
Study table has some values 'NULL' instead of NULL
ENSVAR-3134

- Changed the SQL queries to insert or update a study
- Found HTML character in study description `&lt;` add to `replace_text`

### CODE REVIEW

File: scripts/import/import_all_sv_data.pl [hunk 1]
```diff
@@ -428,62 +427,45 @@ sub study_table{
...
-        $external_link_sql = '';
-      }
-      else {
-        $external_link_sql = "external_reference='$external_link',";
-      }
-    }
-    else {
-      $external_link_sql = "external_reference='$external_link',";
+    if ($external_link && $external_link eq 'NULL' && $external_link eq '') {
+      $external_link = undef;
```
Comment (sarahhunt) [inline scripts/import/import_all_sv_data.pl:431]: should this be null or empty?

File: scripts/import/import_all_sv_data.pl [hunk 2]
```diff
@@ -428,62 +427,45 @@ sub study_table{
...
+                  );
 
-    $dbVar->do($stmt);
   }
   # INSERT
   else {
-    if ($external_link && $external_link ne 'NULL' && $external_link ne '') {
-      $external_link = "'$external_link'";
+    if($external_link && $external_link eq 'NULL') {
+      $external_link = undef;
```
Comment (sarahhunt) [inline scripts/import/import_all_sv_data.pl:446]: this is similar to L430 - can nulls be dealt with once per input line?

Comment (dglemos) [reply scripts/import/import_all_sv_data.pl:446]: Good point. I'll do this before the if .. else

File: scripts/import/import_all_sv_data.pl (hunk 1 above)
Comment (dglemos) [reply scripts/import/import_all_sv_data.pl]: It didn't make sense, it's fixed now.

**Pull request merged**